settings.gradle

```diff
@@ -1,3 +1,10 @@
+pluginManagement {
+    repositories {
+        google()
```

Comment:
required, otherwise the plugin is unable to locate `com.android.tools.build:gradle:7.1.3`, it tries to find it within the gradle plugin portal only